disk/disk_windows.go

```diff
@@ -32,6 +32,15 @@ type Win32_PerfFormattedData struct {
 	AvgDisksecPerRead       uint64
 	AvgDisksecPerWrite      uint64
 }
+type Win32_DiskDrive struct {
+	DeviceID         string
+	FirmwareRevision string
+	InterfaceType    string
+	SerialNumber     string
+}
+type Win32_DiskPartition struct {
```

Comment:
Please don't export these types (I know there are similarly exported wmi types in other gopsutil packages but that's a mistake as Windows-only and thus should not be part of the public API).